Dockerfile.windows

```diff
@@ -153,7 +153,7 @@
 
 
 # The number of build steps below are explicitly minimised to improve performance.
-FROM microsoft/windowsservercore
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
No, no, no, no!!!! This is a common dockerfile for many different builds (including nightly internally). It is expected that "microsoft/windowsservercore:latest" exists when building this. Changing this from as-is will break RS1 CI and more. Please revert this change. Thanks :)

Reply:
Hello @jhowardmsft thanks for the heads up, but this is form my WIP commit which is a `git add .` not to lose anything when I closing the day, I will revert it when I am back, sorry for the hassle caused.